README.md

```diff
@@ -2,36 +2,26 @@
 We at [@SchweizerischeBundesbahnen](https://github.com/SchweizerischeBundesbahnen) need to host all the OpenShift events outside our OSE-cluster as it would flood our etcd datastore if we keep the events of all projects for more than one day.
 So this tools just attaches to the kubernetes API and logs all the events to console where they are grabbed and sent to our central logging environment.
 
+# Syslog forwarding
```

Comment:
Describe that the standard mode is "Stdout" and optional there is the "syslog" mode. Otherwise people could be confused :)